scripts/publish_to_s3.py

```diff
@@ -0,0 +1,69 @@
+import os
```

Comment:
I think probably we should either make this an importable / runnable module which can be invoked with something like `python -m werkit.publish_to_s3.cli`, or else ship a bin program with the module which will get installed in the path.

The first approach is easy and would mean moving this file into the werkit namespace.

I haven't done the second approach with Poetry (at least I don't think I have) but found this article about it and would be happy to dig into it if you want:

https://dev.to/bowmanjd/build-command-line-tools-with-python-poetry-4mnc

Also, we should add a build extra for this publishs cript which requires boto and click.

Reply:
I moved this file into the werkit namespace, and here is how I am now invoking it from anatomia CI:

```bash
            $( python3 -c "import werkit.scripts.publish_to_s3; print(werkit.scripts.publish_to_s3.__file__)" ) \
              publish-using-sha1 \
              --function-name anatomia \
              --bucket-name purl-zip-archive \
              --with-manifest
```

https://github.com/metabolize/purl-anatomia/pull/446/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R215

Let me know what you think about this pattern.

Reply:
The script invocation now looks like this:

```
            python -m werkit.scripts.publish_to_s3 \
              infer-and-publish \
```